src/uu/stty/src/stty.rs

```diff
@@ -318,149 +290,125 @@ fn stty(opts: &Options) -> UResult<()> {
-                    AllFlags::InputFlags((flag, remove)) => check_flag_group(flag, remove),
-                    AllFlags::LocalFlags((flag, remove)) => check_flag_group(flag, remove),
-                    AllFlags::OutputFlags((flag, remove)) => check_flag_group(flag, remove),
-                };
-                if remove_group {
-                    return Err(USimpleError::new(
-                        1,
-                        get_message_with_args(
-                            "stty-error-invalid-argument",
-                            HashMap::from([("arg".to_string(), (*arg).to_string())]),
-                        ),
-                    ));
-                }
-                valid_args.push(flag.into());
-            // combination setting
-            } else if let Some(combo) = string_to_combo(arg) {
-                valid_args.append(&mut combo_to_flags(combo));
-            } else if arg == "rows" {
-                if let Some(rows) = args_iter.next() {
-                    if let Some(n) = parse_rows_cols(rows) {
-                        valid_args.push(ArgOptions::Special(SpecialSetting::Rows(n)));
+                "columns" | "cols" => {
+                    if let Some(cols) = args_iter.next() {
+                        if let Some(n) = parse_rows_cols(cols) {
+                            valid_args.push(ArgOptions::Special(SpecialSetting::Cols(n)));
+                        } else {
+                            return invalid_integer_arg(cols);
+                        }
                     } else {
-                        return Err(USimpleError::new(
-                            1,
-                            get_message_with_args(
-                                "stty-error-invalid-integer-argument",
-                                HashMap::from([("value".to_string(), format!("'{rows}'"))]),
-                            ),
-                        ));
+                        return missing_arg(arg);
                     }
-                } else {
-                    return Err(USimpleError::new(
-                        1,
-                        get_message_with_args(
-                            "stty-error-missing-argument",
-                            HashMap::from([("arg".to_string(), (*arg).to_string())]),
-                        ),
-                    ));
                 }
-            } else if arg == "columns" || arg == "cols" {
-                if let Some(cols) = args_iter.next() {
-                    if let Some(n) = parse_rows_cols(cols) {
-                        valid_args.push(ArgOptions::Special(SpecialSetting::Cols(n)));
+                "drain" => {
+                    set_arg = SetArg::TCSADRAIN;
+                }
+                "-drain" => {
+                    set_arg = SetArg::TCSANOW;
+                }
+                "size" => {
+                    valid_args.push(ArgOptions::Print(PrintSetting::Size));
+                }
+                &_ => {
```

Comment:
What's the reason for the `&`? I think it is not necessary:
```suggestion
                _ => {
```